### PR TITLE
Update cs _common strings

### PR DESCRIPTION
**Pull request by zenhas**
Added czech translation for several lists and  expansion_rule strings.

### CODE REVIEW

File: sentences/cs/_common.yaml [hunk 1]
```diff
@@ -7,6 +7,58 @@ responses:
...
+  name: "{name}"
+  area: "{area}"
+  what_is: "jak(á|é|ý) je"
+  how_many: "kolik je"
+  brightness: "{brightness} [percent]"
+  temp: "(teplota)"
+  temperature: "{temperature} [stupňů] [{temperature_unit}]"
+  turn_on: "zapni"
+  turn_off: "vypni"
+  turn_on_light: "(rozsviť|rozsviť)"
```
Comment (halecivo) [inline sentences/cs/_common.yaml:59]: two same values here

File: sentences/cs/_common.yaml [hunk 2]
```diff
@@ -7,6 +7,58 @@ responses:
...
+        out: "Fahrenheit"
+expansion_rules:
+  name: "{name}"
+  area: "{area}"
+  what_is: "jak(á|é|ý) je"
+  how_many: "kolik je"
+  brightness: "{brightness} [percent]"
+  temp: "(teplota)"
+  temperature: "{temperature} [stupňů] [{temperature_unit}]"
+  turn_on: "zapni"
```
Comment (halecivo) [inline sentences/cs/_common.yaml:57]: This is already done as part of PR #237, please check existing PRs before starting a new one.
If you have different idea for the words, it should be discussed there.

File: sentences/cs/_common.yaml [hunk 3]
```diff
@@ -7,6 +7,58 @@ responses:
...
+  temperature:
+    range:
+      type: "temperature"
+      from: 0
+      to: 100
+  temperature_unit:
+    values:
+      - "celsius"
+      - in: "c"
+        out: "Celsius"
```
Comment (halecivo) [inline sentences/cs/_common.yaml:45]: Shouldn't this be rather Celsia, since we sat "18 stupňů Celsia"?

Comment (zenhas) [reply sentences/cs/_common.yaml:45]: Sorry, don't know how to update the patch in pull request.

Yes "Celsia" is better

File: sentences/cs/_common.yaml (hunk 1 above)
Comment (zenhas) [reply sentences/cs/_common.yaml:59]: orry, overlooked that.

Should be:

turn_off: "(vypni|vypnout|zhasni|zhasnout)"
turn_on_light: "(zapni|zapnout|rozsviť|rozsvítit)"

As I told, I don' know how to update PR patch from.

**Comment by halecivo** [comment]
Hello @zenhas,
 - you can now update your branch on latest main, because my PR has been merged, do you know how to do that? 
 - I learned that expansion rules, which are not slots, should be in native language. I use infinite form for verbs, <zapnout> should be used instead of <turn_on>. 
 - Trivial expansion rules can be removed, like <name> expanding to "{name}".

File: sentences/cs/_common.yaml (hunk 3 above)
Comment (TomaszPilch) [reply sentences/cs/_common.yaml:45]: @zenhas I think you should create the PR from branch in your fork, not just from commits https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork

Comment (zenhas) [reply sentences/cs/_common.yaml:45]: @TomaszPilch  Will do in the future. 
Thought that when i make fork and work on is enought. I created PR from my forked master. Anyway, next time I'll make branch and request merge from that.
Thanks.

**Comment by zenhas** [comment]
I think, we can drop this PR as it brings nothing new. Changes I made were merged by another PR #237

**Pull request closed without merging**